generated/nidaqmx/_library_interpreter.py

```diff
@@ -1758,12 +1754,14 @@ def create_table_scale(
                         ctypes_byte_str]
 
         error_code = cfunc(
-            name, prescaled_vals, len(prescaled_vals), num_prescaled_vals_in,
-            scaled_vals, len(scaled_vals), num_scaled_vals_in,
-            pre_scaled_units, scaled_units)
+            name, prescaled_vals, len(prescaled_vals), scaled_vals,
+            len(scaled_vals), pre_scaled_units, scaled_units)
         check_for_error(error_code)
 
     def create_task(self, session_name):
+        task = lib_importer.task_handle(0)
+        task = ctypes.TaskHandle()
```

Comment:
```
>>> task=interpreter.create_task("MyTask")
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "D:\dev\nidaqmx-python\generated\nidaqmx\_library_interpreter.py", line 1763, in create_task
    task = ctypes.TaskHandle()
AttributeError: module 'ctypes' has no attribute 'TaskHandle'
```

Reply:
This problem has been fixed and the unwanted instantiation has been removed